blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/RunContainerImpl.java

```diff
@@ -59,6 +80,23 @@ public BlueRun get(String name) {
         return  AbstractRunImpl.getBlueRun(run, pipeline);
     }
 
+    /**
+     * Finds a run in the queue based on the expectedBuildNumber, returns the queue id if found
+     */
+    private String findRunInQueue(String name) {
```

Comment:
again, it's finding something in a different place. it's finding a `run` in the list of `queue` items.